python/interpret_text/common/preprocessor/bert_preprocessor.py

```diff
@@ -0,0 +1,82 @@
+import pandas as pd
+from typing import Any, Iterable
+
+from interpret_text.common.base_explainer import BaseTextPreprocessor
+from transformers import BertTokenizer
+
+
+class BertPreprocessor(BaseTextPreprocessor):
+    
+    MAX_TOKEN_LENGTH = 50  # change to param?
+    TOKEN_PAD_TO_MAX = True 
+    BUILD_WORD_DICT = True  # Default = False?
+    
+    def build_vocab(self, text: str) -> Any:
+        words_to_indexes = {}
+        counts = {}
+        tokenizer = self.get_tokenizer()
+        
+        for special_token in tokenizer.all_special_tokens:
+            words_to_indexes[special_token] = tokenizer.convert_tokens_to_ids(special_token)
+            counts[special_token] = 1000
```

Comment:
what does this do - why does it set it to 1k?  also maybe use constant in place of 1k (aka magic numbers)

Reply:
Hmm not sure why it's set to 1k since it was in the existing code. Noted this and will get back when i understand better.